### PR TITLE
Allow creating sample sheets for pools

### PR DESCRIPTION
**Pull request by apmasell**
JIRA: GLT-2902

### CODE REVIEW

**Comment by avarsava** [comment]
CI is Very Mad about javascript

**Comment by apmasell** [comment]
I did some formatting that went not so good.

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java [hunk 1]
```diff
@@ -509,4 +540,16 @@ public LibraryAliquotDto asDto(LibraryAliquot model) {
     return parentFinder.list(ids, category);
   }
 
+  @PostMapping(value = "/samplesheet/{format}")
+  @ResponseBody
+  public HttpEntity<byte[]> samplehsheet(@PathVariable("format") String format, @RequestBody SampleSheetRequest request,
```
Comment (avarsava) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java]: samplehsheet?

Comment (apmasell) [reply miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java]: Fixed.

**Pull request merged**